File: 250218/숫자로 이루어진 사각형/rectangle-with-a-number.java
import java.util.*;
import java.io.*;

public class Main {
    public static void printRect(StringBuilder sb, int n) {
        int cnt = 1;
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < n; j++) {
                sb.append(cnt).append(" ");
                cnt = cnt % 9 + 1;
            }
            sb.append("\n");
        }
    }

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();

        int n = Integer.parseInt(br.readLine());
        printRect(sb, n);
        System.out.println(sb);
    }
}

File: 250218/숫자로 이루어진 사각형/rectangle-with-a-number.cpp
#include <iostream>

using namespace std;

void PrintRect(int n) {
    int cnt = 1;
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            cout << cnt << " ";
            cnt = cnt % 9 + 1;
        }
        cout << "\n";
    }
}

int main() {
    int n;

    cin >> n;
    PrintRect(n);
    return 0;
}